.evergreen/run-orchestration.sh

```diff
@@ -117,6 +134,9 @@ if [ -z "${SKIP_CRYPT_SHARED:-}" ]; then
     fi
   cat <<EOT >> mo-expansion.yml
 CRYPT_SHARED_LIB_PATH: "$CRYPT_SHARED_LIB_PATH"
+EOT
+cat <<EOT >> mo-expansion.sh
+export CRYPT_SHARED_LIB_PATH="$CRYPT_SHARED_LIB_PATH"
```

Comment:
```suggestion
export CRYPT_SHARED_LIB_PATH
```
Assignment not necessary?

Reply:
This is creating the equivalent of a `.env` file that we can source, as a local alternative to expansion files.